singularity/registration-toolkit.def

```diff
@@ -0,0 +1,68 @@
+Bootstrap: docker
+From: ubuntu:21.04
+
+%labels
+  MAINTAINER Stephen Parsons <stephen.parsons@uky.edu>
+  OS Ubuntu 21.04
+
+%files
+  /tmp/registration-toolkit /usr/local/educelab/registration-toolkit
+
+%post
+  # Give read/write access in registration-toolkit (so it can later be edited using a persistent overlay)
+  chmod --recursive a+rw /usr/local/educelab/registration-toolkit
+  # Basic installs (noninteractive so tzdata dependency install doesn't halt build with prompt)
+  apt update
+  DEBIAN_FRONTEND=noninteractive apt install -y git nano vim build-essential
+  # Store git credentials for up to an hour (in overlay, not container) for convenience
+  git config --global credential.helper cache
+  git config --global credential.helper 'cache --timeout=3600'
+  # Install registration-toolkit dependencies
+  apt install -y \
+    cmake \
+    libopencv-dev \
+    libvtk6-dev \
+    libboost-program-options-dev \
+    libinsighttoolkit4-dev
+  # Build registration-toolkit
+  cd /usr/local/educelab/registration-toolkit
+  mkdir build && cd build
+  cmake ..
+  make -j$(nproc --all)
+  # Install registration-toolkit
+  make install
```

Comment:
Do we want to remove the build directory since everything is installed?

Reply:
Doesn't matter to me. Sure